db/database.py

```diff
@@ -0,0 +1,48 @@
+from motor.motor_asyncio import AsyncIOMotorClient,AsyncIOMotorCollection
+from bson.objectid import ObjectId
+from dotenv import load_dotenv
+import os
+from fastapi.encoders import jsonable_encoder
+url = os.getenv("DB_URL")
```

Comment:
Should store this in settings.py, this file should include all function for database not get env from settings bro